data/local.yaml

```diff
@@ -1089,3 +1089,11 @@ S21F:
 S22F:
   description: Time-based milestone marking the completion of the Spring 2022 development
     cycle.
+LDM-GEN3:
+  description: 'Show parity of Butler Gen3 with Butler Gen2 systems.
+   We should be able to run processing pipelines with Gen3 which previously
```

Comment:
I'd suggest a stronger phrasing, like "Demonstrate the ability to run under Gen3 key DRP and AP pipelines that previously ran on Gen2."

Reply:
I think its ok